wandb/sdk/internal/sender.py

```diff
@@ -1394,6 +1412,29 @@ def get_local_info(self) -> "LocalInfo":
             )
         return local_info
 
+    def _flush_job(self) -> None:
+        self._job_builder.set_config(
+            {k: v for k, v in self._consolidated_config.items() if k != "_wandb"}
+        )
+        summary_dict = self._cached_summary.copy()
+        summary_dict.pop("_wandb", None)
+        self._job_builder.set_summary(summary_dict)
+        if (
+            not hasattr(self._settings, "_offline")
```

Comment:
just means that staticsettings doesn't have `_offline` field, probably better just to add it instead of this condition?

Reply:
Ah, I originally did that, but thought maybe there was a reason for not including it. Will change

Reply:
nope, it is just that the static setting object is not automatically generated as we add fields to the `wandb.Setttings` and they go out of sync

Reply:
I've switched to the more canonical get function to fetch this value. Offline exists optionally, this seems like the most correct solution.

Reply:
hmmm... it is weird that `_offline` is optional... i think it should be just boolean... 
look at the wandb_settings file it seems like it always assigned a value...